tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -47,4 +55,49 @@ public function emptyValueProvider()
             'false' => [false, true]
         ];
     }
+
+    /**
+     * @dataProvider priceValueProvider
+     *
+     * @param $value string|int value that should be checked.
+     * @param $shouldCauseException bool should an exception be caused by given parameter.
+     */
+    public function testPriceValueDetectsNumericsOnly($value, $shouldCauseException)
+    {
+        try {
+            $price = new Price();
+            $price->setValue($value);
+
+            if ($shouldCauseException) {
+                $this->fail('Should be detected as numeric value.');
+            } else {
+                // The following assertion exists mostly to ensure that PHPUnit does not lament
+                // the lack of assertions in this successful test.
+                $this->assertNotNull($price);
+            }
+        } catch (ValueIsNotNumericException $e) {
+            if (!$shouldCauseException) {
+                $this->fail('Should not be detected as numeric value.');
+            } else {
+                $this->assertEquals('Value is not a valid number!', $e->getMessage());
+            }
+        }
+    }
+
+    /**
+     * Scenarios for numeric price value validation.
+     *
+     * @return array Cases with the value to check and whether it should cause a validation issue.
+     */
+    public function priceValueProvider()
```

Comment:
Rename provider to match the correct meaning of the test

Reply:
✅